ssg/build_yaml.py

```diff
@@ -1024,6 +1024,7 @@ class Rule(object):
     }
 
     PRODUCT_REFERENCES = ("stigid", "cis",)
+    GLOBAL_REFERENCES = ("srg", "disa",)
```

Comment:
`cis-csc` would be another one here that could be added. It has no product-specific uses right now.

Reply:
added